### ISSUE TITLE
Support of JSR-340 & JSR-356

### ISSUE THREAD

**Issue by fomkin** [fomkin/korolev#63]
It would be cool to support Java EE application containers.

**Comment by zhan-ge** [comment]
Any progress here? Or just provide a simple tutorial about how to implement custom server for korolev, I would like to integrate korolev with spring webflux.

**Comment by fomkin** [comment]
> Any progress here?

No, but you can try [http4s-servlet](https://medium.com/@takezoe/http4s-on-the-servlet-container-436dba2f142e) using [korolev-http4s](https://fomkin.org/korolev/user-guide.html#_http4s) interop.

> Or just provide a simple tutorial about how to implement custom server for korolev

You can rely on existed interop modules.
- https://github.com/fomkin/korolev/blob/master/interop/akka/src/main/scala/korolev/akka/package.scala
- https://github.com/fomkin/korolev/blob/master/interop/http4s/src/main/scala/korolev/http4s/package.scala
